openedx/core/djangoapps/user_api/api.py

```diff
@@ -213,8 +213,9 @@ def _apply_third_party_auth_overrides(request, form_desc):
 
 class RegistrationFormFactory(object):
     """HTTP end-points for creating a new user. """
-
-    DEFAULT_FIELDS = ["email", "name", "username", "password"]
+    # To remove full name field from the registration form shown to the user
+    # the "name" field has been removed from DEFAULT_FIELDS
```

Comment:
Can't we do this through REGISTERATION_FIELDS present in env.json?

Reply:
Actually, the **name** field is not included in the fields that can be controlled through  **env.json** 
```
"REGISTRATION_EXTRA_FIELDS": {
        "city": "hidden",
        "confirm_email": "hidden",
        "country": "optional",
        "gender": "optional",
        "goals": "optional",
        "honor_code": "optional",
        "level_of_education": "hidden",
        "mailing_address": "hidden",
        "terms_of_service": "hidden",
        "year_of_birth": "optional"
    },
```